content/_data/upgrades/2-361-1.adoc

```diff
@@ -0,0 +1,139 @@
+==== Jenkins requires Java 11 or newer
+
+Beginning with Jenkins 2.357, which was released on June 28, 2022, Jenkins requires Java 11 or newer on both the controller JVM and agent JVMs.
+
+Prior to Jenkins 2.357 and Jenkins 2.361.1, running the controller on Java 11 and agents on Java 8, though not recommended, did not result in errors.
+Beginning with Jenkins 2.357 and Jenkins 2.361.1, running the controller on Java 11 and agents on Java 8 will result in the following error:
+
+[source]
+----
+Error: A JNI error has occurred, please check your installation and try again
+Exception in thread "main" java.lang.UnsupportedClassVersionError: hudson/remoting/Launcher has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
+	at java.lang.ClassLoader.defineClass1(Native Method)
+	at java.lang.ClassLoader.defineClass(ClassLoader.java:756)
+	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
+	at java.net.URLClassLoader.defineClass(URLClassLoader.java:473)
+	at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
+	at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
+	at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
+	at java.security.AccessController.doPrivileged(Native Method)
+	at java.net.URLClassLoader.findClass(URLClassLoader.java:362)
+	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
+	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
+	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
+	at sun.launcher.LauncherHelper.checkAndLoadMain(LauncherHelper.java:601)
+----
+
+Therefore, it is critical to upgrade both the controller _and_ agents to Java 11 or newer prior to upgrading Jenkins to 2.357 or 2.361.1.
+Use the link:https://plugins.jenkins.io/versioncolumn/[Versions Node Monitors] plugin to verify that agents are running a compatible version of Java.
+
+The official Jenkins Docker images for link:https://hub.docker.com/r/jenkins/jenkins/[the controller] and link:https://hub.docker.com/r/jenkins/inbound-agent/[agents] have been based on Java 11 for many months, with Java 8 available as a fallback, and Java 17 available in preview mode.
+Beginning with the weekly release line of Jenkins 2.357, the Java 8 images have been retired and the Java 17 images have transitioned from preview to general availability.
+Users of the official Docker images do not need to install or configure Java on their own, as it comes preinstalled in the Docker images.
+
+If your application build still requires Java 8, and you are using a Docker image to run the agent Java process simultaneously, such as `remoting.jar`, you will need to provide a Java 11 or newer runtime for the Jenkins agent process and a Java 8 environment for your application build.
```

Comment:
Most of the copyedits are fine, but this one seems worse than the original. I originally wrote "i.e., `remoting.jar`", which is an abbreviation of the Latin _id est_, which means "that is", which is used to introduce or follow a clarification. "Such as" means the same thing as "for example", implying that there are other examples. In fact, there are no other such examples, since `remoting.jar` (i.e., Remoting) is the only possible example of what the agent Java process could be running. So the literal translation "that is" is more logically accurate than "such as".

Reply:
Thanks for the insight on this one Basil, I wasn't aware that this was the only example of what could be done. I've revised the sentence structure to have it make more sense. In terms of the i.e. usage, it is something that the docs team tries to avoid, so I was trying to convey the precision. Knowing that it's the only process/example makes a big difference, so thank you again!